controllers/sms-gateway.js

```diff
@@ -54,23 +54,24 @@ function updateState(messageId, newState, details, callback) {
   messageUtils.updateMessage(messageId, updateBody, callback);
 }
 
-// function markMessagesForwarded(messages, callback) {
-//   async.eachSeries(
-//     messages,
-//     function(message, callback) {
-//       updateState(message.id, 'forwarded-to-gateway', null, callback);
-//     },
-//     function(err) {
-//       if (err) {
-//         warn(err);
-//       }
-//       return callback();
-//     }
-//   );
-// }
+function markMessagesForwarded(messages, callback) {
+  async.eachSeries(
+    messages,
+    function(message, callback) {
+      updateState(message.id, 'forwarded-to-gateway', null, callback);
+    },
+    function(err) {
+      if (err) {
+        warn(err);
```

Comment:
It might be worth a comment here to explain why you're ok with just warning on `err`.